lib/al/Library/Play/Camera/CameraVerticalAbsorber.h

```diff
@@ -59,5 +54,4 @@ class CameraVerticalAbsorber : public al::NerveExecutor {
     void tryResetAbsorbVecIfInCollision(const sead::Vector3f&);
     void update();
 };
-static_assert(sizeof(CameraVerticalAbsorber) == 0x1b0, "blaah!");
```

Comment:
Why would you remove this check?

Reply:
It's not done on many classes and it's just a remnant from an old repo. In any case, the chance that the size of the class accidentally changing should be next to none and this shouldn't be required.

Reply:
We can add static asserts in another PR/commit if that's wanted. I am not adding it back in this PR

Reply:
I'd insist on not removing this check in the PR, to avoid the effort of adding it back later.

Reply:
Sure. I'll add it back and fix it up so its `<class> size mismatch` like the sead asserts then